frontend/src/cards/ui/HighestLowestList.tsx

```diff
@@ -104,8 +106,13 @@ export function HighestLowestList(props: HighestLowestListProps) {
       <p>All rates are reported as: {props.metricConfig.fullCardTitleName}</p>
       <p>
         Consider the possible impact of{" "}
-        <a href="#missingDataInfo">data reporting gaps</a> when interpreting the
-        highest and lowest rates.
+        <Button
+          onClick={() => props.jumpToData!()}
```

Comment:
Maybe better to account for situations where the jumpTo method hasn’t been passed; or require it as a prop?